ops/start-chain.sh

```diff
@@ -36,7 +36,7 @@ then
   then version="`echo $git_tag | sed 's/indra-//'`"
   else version="`git rev-parse HEAD | head -c 8`"
   fi
-  image="${project}_ethprovider:$version"
+  image="connextproject/${project}_ethprovider:$version"
```

Comment:
Check the top of this file & use that `registry` variable instead of hard-coding it here